pkg/kudoctl/cmd/package_params_list.go

```diff
@@ -56,7 +56,7 @@ func newParamsListCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	f.BoolVarP(&list.requiredOnly, "required", "r", false, "Show only parameters which have no defaults but are required.")
 	f.BoolVar(&list.namesOnly, "names", false, "Display only names.")
 	f.StringVar(&list.RepoName, "repo", "", "Name of repository configuration to use. (default defined by context)")
-	f.StringVarP(&list.PackageVersion, "version", "v", "", "A specific package version on the official GitHub repo. (default to the most recent)")
+	f.StringVarP(&list.PackageVersion, "version", "", "", "A specific package version on the official GitHub repo. (default to the most recent)")
```

Comment:
@sharego, thx for the fix! You can just remove shortHand parameter altogether:
```suggestion
	f.StringVarP(&list.PackageVersion, "version", "", "A specific package version on the official GitHub repo. (default to the most recent)")
```

Reply:
Yes, that is correct, that fix is the right one, you need to go from StringVarP to StringVar